CONTRIBUTING.md

```diff
@@ -1,135 +1,96 @@
-# Contributing
+# Contributing Guidelines
 
-## Getting Started
+Thank you for your interest in contributing to our project! 💛
 
-1. Fork & Clone this repo
-1. [`nvm install`](https://github.com/nvm-sh/nvm)
-1. [`nvm use`](https://github.com/nvm-sh/nvm)
-1. `yarn install`
-1. `yarn build`
-
-## Documentation Development
-
-1. Run the documentation via `yarn docs dev`
-1. Visit <http://localhost:5000/>
-1. Create/Update content based on the URL.
-
-   For example, the content for
-   http://localhost:5000/components/authenticator is located at [`docs/src/pages/components/authenticator/index.mdx`](docs/src/pages/components/authenticator/index.mdx)
-
-Internally, this content is served by a single, Next.js [optional catch all route](https://nextjs.org/docs/routing/dynamic-routes#optional-catch-all-routes):
-[`docs/src/pages/[[...slugs]].tsx`](docs/src/pages/[[...slugs]].tsx).
-
-## `@aws-amplify/ui`
-
-- `yarn ui build` to rebuild for production.
-- `yarn ui dev` to watch & rebuild for development.
-
-## `@aws-amplify/ui-react`
-
-- `yarn react build` to rebuild for production.
-- `yarn react dev` to watch & rebuild for development.
-- `yarn next-example dev` to run React examples on Next.js.
-
-## `@aws-amplify/ui-vue`
-
-- `yarn vue build` to rebuild for production
-- `yarn vue dev` to watch & rebuild for development.
-- `yarn vue-example dev` to run Vue examples on Vite.
-
-## `@aws-amplify/ui-angular`
-
-- `yarn angular build` to rebuild for production
-- `yarn angular dev` to watch & rebuild for development
-- `yarn angular-example dev` to run Angular examples.
-
-### Angular Troubleshooting
+Please read through these guidelines carefully before submitting a PR and let us know if it's not up to date (or even better, submit a PR with your corrections 😉).
 
-Whenever `yarn angular build` is run, such as part of postinstall script, you want to run `yarn angular dev` again. This is to ensure that both `yarn angular` and `yarn angular-example` have [ivy](https://angular.io/guide/ivy) enabled for local development.
+## Table of Contents
 
-## Examples Development
+- [Bug Reports](#bug-reports)
+- [Pull requests](#getting-started)
```

Comment:
The `Pull requests` link does not work. Should it be `#pull-requests` ?

Reply:
Also probably should be `Pull Requests`

Reply:
Good catch 🙏